### ISSUE TITLE
Update brlapi bindings to support Python 3.13 and X64

### ISSUE THREAD

**Issue by LeonarddeR** [nvaccess/nvda#18657]
### Is your feature request related to a problem? Please describe.
brlapi bindings for BRLTTY should be updated to support Python 3.13 and X64 builds of NVDA.

_Originally posted by @seanbudd in https://github.com/nvaccess/nvda/issues/18611#issuecomment-3157693156_

> We've reached out to Dave and other from the project a couple times and haven't heard back yet, but we are hoping to get a range of builds (32/64, 3.11/3.13) that we can use in the transition. At the very least, we need the 3.13 and 64 bit build.

It makes sense to have an issue to discuss this and track progress.

### Describe the solution you'd like
Have brlapi builds for Python 3.13 and X64 support.

### Additional context
I performed some research. Unfortunately it is not really hopeful.

1. Creating a Python 3.13 X86 build is no big deal when you setup MSYS1 according to the docs
2. Creating an X64 build of BRLTTY, brlapi and/or the bindings is going to be problematic. Windows itself bundles an x86 version of brlapi when installing Narrator support. This was observed when trying to load "C:\Windows\brltty\bin\brlapi-0.8.dll" in an X64 application.

@Bramd once investigated building brltty with MSYS2. This should be possible, but MSYS2 bundles its own Python interpreter which is currently at 3.12 and extensions build with msys2 Python aren't compatible with vanilla Python.

**Comment by LeonarddeR** [comment]
Mainly because of the great work by @bramd, I started porting his Github Actions work to MSYS2 UCRT64. Locally, I've been able to compile BRLTTY X64 and even create a python 3.13 pyd file (not yet tested).
There is light at the end of the tunnel.

**Comment by LeonarddeR** [comment]
After a day of fiddling, thanks to preparatory work by @bramd, I think I was able to build something that works. See https://github.com/brltty/brltty/pull/490

**Comment by LeonarddeR** [comment]
For reference, https://github.com/brltty/brltty/pull/490 was prematurely closed.
BRLTTY's code of conduct doesn't seem to be compatible with mine. This gives me too much frustration and therefore I"m taking off my hands of current and future BRLTTY related work. At least the NV Access fork is able to build it and will hopefully be able to do it in the near and more distant future.
